contracts/v2/PolygonRollupManager.sol

```diff
@@ -531,8 +524,12 @@ contract PolygonRollupManager is
         uint32 rollupTypeID = ++rollupTypeCount;
 
         if (rollupVerifierType == VerifierType.Pessimistic) {
-            // No genesis on pessimistic rollups
-            if (genesis != bytes32(0)) revert InvalidRollupType();
+            // Those params should be zero for Pessimistic rollup types
+            if (
+                genesis != bytes32(0) ||
```

Comment:
`forkID` also to be 0.
Check are the same as `ALGateway`. Therefore, it could be done together.